Geocortex.Mobile.Samples/Geocortex.Mobile.Samples.UWP/Geocortex.Mobile.Samples.UWP.csproj

```diff
@@ -17,16 +17,6 @@
     <EnableDotNetNativeCompatibleProfile>true</EnableDotNetNativeCompatibleProfile>
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
```

Comment:
The `ProjectGuid` is interesting @kewalker. Not sure how this is used by the various platforms.

Reply:
Hmm - that is interesting. This seems to be the same Guid between the samples and quickstart. I think the recommendation is that they are unique. I think these Guids are more used for project references, so you might have issues if you tried to reference the same colliding Guid projects. I don't think this will cause issues during deployment, as I had both of these projects installed side-by-side on my machine. Also, it's worth noting that `ProjectTypeGuids` are expected to be non-unique.